docs/user/subprojects.rst

```diff
@@ -46,6 +46,18 @@ Adding a custom domain to a subproject is not allowed,
 since your documentation will always be served from
 the domain of the parent project.
 
+Separate release cycles
+-----------------------
+
+When several projects are documented in the same space, it's common that they follow separate release cycles.
+
+We recommend that the documentation follows the release cycle of whatever it is documenting.
+By using subprojects, you can present your documentation for several projects with separate release cycles.
+This is solved by having the :term:`flyout menu` active for the project that's viewed.
+
+When the user navigates to a subproject,
+they are presented with a :term:`flyout menu` matching the subproject's versions and :doc:`/downloadable-documentation`.
```

Comment:
Weird to use the `term` twice in such quick succession I think.

Overall I'm not quite sure what this whole section is trying to say? I think an example or screenshot could help, but I don't get the value it's trying to show? That there's a flyout, and it exists?

Reply:
I've updated it now. Does the point stand out clearer? I think the point is very critical to make because it's definitely one of the reasons that I think this feature is very useful (and because I think _not_ supporting separate release cycles would render it fairly useless).